sklearn/cluster/_optics.py

```diff
@@ -232,11 +232,12 @@ def fit(self, X, y=None):
         Parameters
         ----------
         X : array, shape (n_samples, n_features), or (n_samples, n_samples)  \
-if metric=’precomputed’.
+if metric=’precomputed’
```

Comment:
This looks pretty odd in the file. I am not sure about conventions in the case where this line is too long.

@amueller

Reply:
I think this is correct. @thomasjpfan can confirm or deny.

Reply:
In general, I prefer not having periods on the type.

Reply:
The question was about the backslash and line length ;)

Reply:
We haven't established a convention. (We should) I tend to prefer:

```py
    X : array, shape (n_samples, n_features), or (n_samples, n_samples)  \
    if metric='precomputed'
        A feature array,....
```

They will render the same.

Reply:
> In general, I prefer not having periods on the type.

That would also raise errors with the docsting validator I think.

+1 for the style in the comment above..